app/services/hyrax/workflow/status_list_service.rb

```diff
@@ -19,13 +19,20 @@ def each
 
       private
 
+        delegate :logger, to: :context
+
+        def user
```

Comment:
The user method is privatized (which I consider a good thing in regards to methods) but it is a change to the interface of the object.